torch/testing/_internal/distributed/nn/api/remote_module_test.py

```diff
@@ -24,6 +25,11 @@ def remote_device(module_rref):
         return param.device
 
 
+# RPC handler for querying __dict__ on the destination worker.
+def remote_module_attributes(remote_module):
```

Comment:
We should also test that we can actually execute the remote_module on the new host by calling forward and forward_async and verifying it returns the appropriate results.

Reply:
Added 2 new tests. However, forward_async cannot be initiated from a new host, because the returned future cannot be pickled.